include/ocpp/v201/ocpp_types.hpp

```diff
@@ -829,14 +1928,16 @@ void from_json(const json& j, VPN& k);
 std::ostream& operator<<(std::ostream& os, const VPN& k);
 
 struct NetworkConnectionProfile {
-    OCPPVersionEnum ocppVersion;
+    OCPPInterfaceEnum ocppInterface;
     OCPPTransportEnum ocppTransport;
-    CiString<512> ocppCsmsUrl;
     int32_t messageTimeout;
+    CiString<2000> ocppCsmsUrl;
```

Comment:
TODO: find out implications for backwards compatibility with 2.0.1

Reply:
Accept